src/components/about.jsx

```diff
@@ -1,13 +1,55 @@
 import React from 'react'
+import {Link} from 'react-router'
 
 export default (props) => (
   <div className='box'>
     <div className='box-header-timeline'>
-      Other page
+      About FreeFeed
     </div>
     <div className='box-body'>
       <div>
-        Here be more dragons
+        <h3>What is FreeFeed.net?</h3>
+        <p>FreeFeed is an <a href="https://github.com/FreeFeed/" target="_blank">open-source</a> social
+        network that enables you to discover and discuss
+        the interesting stuff your friends find on the web.</p>
+
+        <p><b><Link to='/signup'>Sign up</Link></b> now or <Link to='/signin'>sign in</Link> if you already have an account.</p>
+
+        <p><img src="/assets/images/sample.png"></img></p>
+
+
+        <h3>Why FreeFeed.net?</h3>
+
+        <p>FreeFeed is being built as a replacement for <a href="https://en.wikipedia.org/wiki/FriendFeed" target="_blank">FriendFeed</a> —
+        the real-time aggregator and social network where 'likes' for user generated content were
+        implemented for the first time.</p>
+
+        <p>After Facebook had acquired FriendFeed and announced its plan to shutdown down the website on April
```

Comment:
"shutdown down".
Also, "announced the plan" sounds better than "announced its plan", I reckon.